composeApp/src/commonMain/kotlin/romHandlers/abstractRomHandlers/AbstractGBRomHandler.kt

```diff
@@ -0,0 +1,12 @@
+package romHandlers.abstractRomHandlers
+
+import configurations.RomConfiguration
+
+abstract class AbstractGBRomHandler(romConfiguration: RomConfiguration,
+                                    protected var rom: ByteArray)
+    : AbstractRomHandler(romConfiguration)
+{
+     override fun saveROM(): ByteArray {
```

Comment:
I get the general idea you were going for here. I don't like that `saveROM()` returns Any. It bypasses the selling point of Java, which is type safety.